app/public/css/styles.css

```diff
@@ -2521,58 +2522,71 @@ li.main-tab input + label.selected:after {
 /* ADDRESSES */
 
 .addresses {
+  width: 100%;
   position: relative;
-  padding-right: 28px;
-}
-
-.addresses::before {
-  position: absolute;
-  content: '';
-  left: 56px;
-  right: 40px;
-  bottom: 40px;
-  height: 56px;
-  background-image: linear-gradient(
-    to top,
-    rgb(255, 255, 255),
-    rgba(255, 255, 255, 0.4)
-  );
-  z-index: 2;
 }
 
 .addresses-title {
   margin-bottom: 24px;
 }
 
 .addresses-content {
-  margin-bottom: 19px;
-  max-height: 736px;
-  overflow-x: hidden;
-  overflow-y: scroll;
-  -webkit-overflow-scrolling: touch;
   position: relative;
 }
 
 .address {
-  padding: 24px 16px 24px 0;
+  padding: 24px 16px 24px 24px;
   border-top: 1px solid var(--color-border);
 }
 
 .address:last-child {
-  margin-bottom: 8px;
+  padding-bottom: 0px;
 }
 
-.address-value {
+.address .value {
+  display: flex;
+  cursor: pointer;
   color: var(--color-grey);
   word-break: break-all;
-  margin-bottom: 16px;
+}
+
+.address.expanded {
+  background-color: var(--color-selected);
+}
```

Comment:
```suggestion
address.expanded {
    border-top: none;
    background-color: var(--color-selected);
    box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.08);
}
```
Just a quick suggestion but I would somehow utilize the box-shadowing to be consistent with other tabs.
![image](https://user-images.githubusercontent.com/22474126/110363929-73dd9e00-8043-11eb-9697-99086086d284.png)

Reply:
But tbh I dont really like that the selected address and the one above are not separated by a line so keeping the `border-top` might be better.

And also this might something to discuss with the designers, especially since the accounts tab has a somewhat similar structure but looks totally different. :smile:

Reply:
Also in the picture the `min-width` for the tabs is set.

Reply:
(Using this thread for more comments since the "global" comments are painful to reply to :smile: )
If possible, I would utilize the accordion behavior we use on `DelegatePage`. Imho that would look really nice.

Reply:
I updated whole behaviour, check it out.